core/src/main/java/org/kohsuke/stapler/export/DataWriter.java

```diff
@@ -69,6 +70,8 @@
     void startObject() throws IOException;
     void endObject() throws IOException;
 
+    @Nonnull ExportConfig getExportConfig();
```

Comment:
Not binary-compatible but I suppose we assume there are no external implementations?